spec/helpers/friends_helper_spec.rb

```diff
@@ -1,12 +1,10 @@
 require 'rails_helper'
 
 RSpec.describe FriendsHelper, type: :helper do
-  let(:friend) { FactoryGirl.create :user }
-  let(:device) do
-    device = FactoryGirl.create(:device, user_id: friend.id)
-    device.checkins << [FactoryGirl.create(:checkin), FactoryGirl.create(:checkin)]
-    device
-  end
+  let(:friend) { create :user }
+  let(:device) { create :device, user: friend }
+
+  before { create_list :checkin, 2, device: device, city: "city" }
```

Comment:
This is a better way to work with associations.
Other cases can be updated accordingly. It's easier to describe explicitly when we do not need association (like [here](https://github.com/earlymarket/Coposition/pull/634/files#diff-1a0461c5944f6bf97b22ff9bb9b63ec7R6)).